tests/migration/migration.go

```diff
@@ -3557,3 +3516,29 @@ func runCommandOnVmiTargetPod(vmi *v1.VirtualMachineInstance, command []string)
 
 	return output, nil
 }
+
+func NewVMWithDataVolumeForMigration(containerDisk cd.ContainerDisk, accessMode k8sv1.PersistentVolumeAccessMode, storageClass string) *v1.VirtualMachineInstance {
```

Comment:
Seems like this function could be kept private.

Reply:
Oops right, will try to remove some of the duplication also. Thanks for the review!